src/com/goide/psi/impl/GoPsiImplUtil.java

```diff
@@ -737,6 +736,30 @@ public boolean value(@NotNull GoMethodSpec spec) {
   }
 
   @NotNull
+  public static List<GoMethodSpec> getAllMethods(@NotNull GoInterfaceType interfaceType) {
+    List<GoMethodSpec> result = ContainerUtil.newSmartList();
+    getAllMethods(interfaceType, result);
+    return result;
+  }
+
+  private static void getAllMethods(@NotNull GoInterfaceType interfaceType, @NotNull List<GoMethodSpec> result) {
```

Comment:
Don't use out parameters, looks like C. Much better to create a function which returns List<GoMethodSpec>.